tox.ini

```diff
@@ -46,7 +46,7 @@ deps =
     sphinx
     sphinx-bluebrain-theme
     sphinx-click
-commands = make html SPHINXOPTS=-W
```

Comment:
this is a useful check to make sure there aren't warnings, please don't remove it.

Reply:
I'm testing atm, I couldn't fix the warning yet...

Reply:
ah, it does not like the :nested: ok... but in the other similar file it works...